ci/homebrew-tap/ci-deploy.sh

```diff
@@ -29,4 +30,10 @@ if [[ $do_push == true ]]; then
   git push origin master
 else
   echo "Refraining from push since it's a dry run"
+  sed -i "s/pypi.io/$pypi_host/" git-machete.rb
+  # install git-machete from local formula with homebrew
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" < /dev/null
+  echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >> /home/circleci/.bash_profile
+  eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+  brew install ./git-machete.rb
```

Comment:
Hmm some run of `git machete`, like checking the version would be useful... unless such test in already executed within the Brew formula itself?